src/components/appmodallogin.main.jsx

```diff
@@ -89,40 +89,28 @@ class AppModalLoginMain extends React.Component {
 
   render() {
     const { failedLogin, isLoading } = this.state;
-
     return (
       <div className="modal login-modal-content" id="login-modal">
         <div className="modal-dialog">
           <div className="modal-content" id="simplemodal-container">
-
-            <div className="modal-header">
-              <h2 className="modal-title">
-                {intl.get('login')}
-              </h2>
-              <button type="button" id="login_modal_close_button" className="close" data-dismiss="modal">
-                &times;
-              </button>
-            </div>
-
-            <div className="feedback-label auth-feedback-container" id="login_modal_auth_feedback_container" data-region="authLoginFormFeedbackRegion" data-i18n="">
-              {failedLogin ? (intl.get('invalid-username-or-password')) : ('')}
-            </div>
-
-            <div className="modal-body">
+            <button type="button" id="login_modal_close_button" className="close" data-dismiss="modal">
+              &times;
+            </button>
+            <div className="form-container">
+              <div className="panel">
+                <h2>
+                  {intl.get('login')}
+                </h2>
+              </div>
               <form id="login_modal_form" onSubmit={this.loginRegisteredUser}>
                 <div className="form-group">
-                  <span>
-                    {intl.get('username')}
-                    :
-                  </span>
-                  <input className="form-control" id="login_modal_username_input" type="text" onChange={this.setUsername} />
+                  <input className="form-control" id="login_modal_username_input" placeholder="Email Address" type="text" onChange={this.setUsername} />
                 </div>
                 <div className="form-group">
-                  <span>
-                    {intl.get('password')}
-                    :
-                  </span>
-                  <input className="form-control" id="login_modal_password_input" type="password" onChange={this.setPassword} />
+                  <input className="form-control" id="login_modal_password_input" placeholder="Password" type="password" onChange={this.setPassword} />
```

Comment:
placeholder text should be localized. There are existing strings in `en-CA.json` that can be used with the following syntax:
`{intl.get('password')}`